### PR TITLE
Update UI theme

### PR DESCRIPTION
**Pull request by muraddurrani**
- Add more of our brand colour into the UI
- Create a logo (not that great so change it if you'd like)

Here is what it looks like:
**Home Page with Scrolling**
home_scroll

**Home Page No Scrolling**
home_no_scroll

**Contact Details Page**
contact details

**Confirm Window**
confirm_window

**Help Window**
help_window

Remove the URL since it cannot be copied to the clipboard anyway

### CODE REVIEW

File: src/main/resources/view/ReacheTheme.css [hunk 1]
```diff
@@ -1,34 +1,40 @@
...
+    -primary_light: #75c7fb;
+    -secondary: #ffcf00;
+    -accent: #ecf1f4;
+    -border: #d4dae6;
+    -background_primary: #fafcfe;
+    -background_secondary: #f4f6f9;
+    -text_primary: #404040;
+    -text_secondary: #555555;
+    -grey: #686F7C;
 }
```
Comment (ckcherry23) [inline src/main/resources/view/ReacheTheme.css:13]: New UI looks great! Can you add the same theme colours in the ConfirmWindow and HelpWindow as well?

**Review by ckcherry23 — COMMENTED** [review]
Looks great

File: src/main/resources/view/ReacheTheme.css (hunk 1 above)
Comment (muraddurrani) [reply src/main/resources/view/ReacheTheme.css:13]: Ah thats a good idea I'll do that

**Review by NicsunXnus — APPROVED** [review]
LG

**Review by ckcherry23 — APPROVED** [review]

**Pull request merged**